docker-compose/docker-compose-esignet/config/esignet-default.properties

```diff
@@ -523,7 +523,7 @@ mosip.esignet.vci.key-values={\
                       'format': 'ldp_vc',\
                       'scope' : 'life_insurance_vc_ldp',\
                       'cryptographic_binding_methods_supported': {'did:jwk'},\
-                      'cryptographic_suites_supported': {'Ed25519Signature2020'},\
+                      'credential_signing_alg_values_supported': {'Ed25519Signature2020'},\
                       'proof_types_supported': {'jwt'},\
```

Comment:
"added background_image credential issuer metadata" - this we should add into our default with a value to showcase this can be supported.

Reply:
Done